### ISSUE TITLE
Docs repo link is missing/inaccessible

### ISSUE THREAD

**Issue by perlun** [gitpod-io/gitpod#20085]
### Bug description

https://github.com/gitpod-io/gitpod/tree/main/docs refers to this:

> The docs content is hosted in the [gitpod-io/website](https://github.com/gitpod-io/website) repository, specifically in Markdown files located at https://github.com/gitpod-io/website/tree/main/gitpod/docs.

...but the `website` repository is not to be found (at least not to me, as a non-Gitpod employee). I get an HTTP 404 trying to access it.

If this is expected, these links ought to be marked with `"(Internal)"` or similar, to make this clear to the wider audience. If the website repo _is_ indeed available, the links should be fixed. If not, please consider making it open in the spirit of open collaboration. :slightly_smiling_face: 

---

_Note to self_: looking to help fix a typo in https://www.gitpod.io/docs/references/ides-and-editors/intellij ("syned").

**Comment by Siddhant-K-code** [comment]
Hey @perlun, Thanks for creating the issue 🧡 

I've addressed your feedback and added the `internal` to the related link (to be resolved in #20087). Also, I've fixed the `syned` typo in IDE docs.

**Comment by perlun** [comment]
Thanks @Siddhant-K-code, looks good. :+1: I approved your change, even though I can of course not formally approve it since I'm not part of the Gitpod team. :slightly_smiling_face: 

I suspected that the website might be internal these days, but good to get it confirmed. I realize this might be asking for too much, but any chance we could make the documentation part of the website non-internal?

I really appreciate project webpages like https://docs.docker.com/desktop/ where you (=anyone, internal or external) can edit the web pages and submit suggestions. Particularly for minimal changes like typos, fixing incorrect URLs and such, it's really useful.

I realize this is probably not for you to decide, and I also don't like what kind of company Gitpod is (Cathedral? Bazaar? Oracle-style? :slightly_smiling_face:), but just wanted to chime in how I personally feel about it. Projects which tend to welcome outside contributions like this automatically "feels nicer" and more "including" to me. :+1:
